Cargo.lock

```diff
@@ -9718,13 +9768,12 @@ dependencies = [
 
 [[package]]
 name = "x25519-dalek"
-version = "2.0.0-rc.2"
+version = "2.0.0-pre.1"
```

Comment:
this looks really weird, but it's because of snow:

https://github.com/mcginty/snow/releases/tag/v0.9.1